app/assets/stylesheets/responsive/_sidebar_layout.scss

```diff
@@ -22,6 +22,12 @@
   }
 }
 
+.new-request-cta {
```

Comment:
it is - becuase moving the follow and report boxes to the action menu made the new-request call to action have way too much space above it